src/Microsoft.AspNetCore.WebHooks.Receivers/Extensions/WebHookReceiverConfigurationExtensions.cs

```diff
@@ -0,0 +1,124 @@
+            }
+            if (sectionKey == null)
+            {
+                throw new ArgumentNullException(nameof(sectionKey));
+            }
+
+            if (string.IsNullOrEmpty(id))
+            {
+                id = WebHookConstants.DefaultIdConfigurationKey;
+            }
+
+            // Look up configuration value for these keys.
+            var key = ConfigurationPath.Combine(
+                WebHookConstants.ReceiverConfigurationSectionKey,
+                sectionKey,
+                WebHookConstants.SecretKeyConfigurationKeySectionKey,
+                id);
+
+            return configuration.GetSection(key);
+        }
+
+        /// <summary>
+        /// Returns an indication WebHook secret key configuration values exist for <paramref name="sectionKey"/>.
+        /// </summary>
+        /// <param name="configuration">The <see cref="IConfiguration"/>.</param>
+        /// <param name="sectionKey">
+        /// The key (relative to <see cref="WebHookConstants.ReceiverConfigurationSectionKey"/>) of the
+        /// <see cref="IConfigurationSection"/> containing the receiver-specific
+        /// <see cref="WebHookConstants.SecretKeyConfigurationKeySectionKey"/> <see cref="IConfigurationSection"/>.
+        /// Typically this is the name of the receiver e.g. <c>github</c>.
+        /// </param>
+        /// <returns>
+        /// <see langword="true"/> if secret key configuration values exist in the receiver-specific
+        /// <see cref="WebHookConstants.SecretKeyConfigurationKeySectionKey"/> <see cref="IConfigurationSection"/>;
+        /// <see langword="false"/> otherwise.
+        /// </returns>
+        public static bool HasWebHookSecretKeys(this IConfiguration configuration, string sectionKey)
+        {
+            if (configuration == null)
+            {
+                throw new ArgumentNullException(nameof(configuration));
+            }
+            if (sectionKey == null)
+            {
+                throw new ArgumentNullException(nameof(sectionKey));
+            }
+
+            var key = ConfigurationPath.Combine(
+                WebHookConstants.ReceiverConfigurationSectionKey,
+                sectionKey,
+                WebHookConstants.SecretKeyConfigurationKeySectionKey);
+
+            return configuration.GetSection(key).Exists();
+        }
+
+        /// <summary>
+        /// Returns <see langword="true"/> if the configuration value with given <paramref name="key"/> is set to
+        /// 'true'; otherwise <see langword="false"/>.
+        /// </summary>
+        /// <param name="configuration">The <see cref="IConfiguration"/>.</param>
+        /// <param name="key">The key of the configuration value to evaluate.</param>
+        /// <returns><see langword="true"/> if the value is set to 'true'; otherwise <see langword="false"/>.</returns>
+        public static bool IsTrue(this IConfiguration configuration, string key)
```

Comment:
>IsTrue [](start = 27, length = 6)

We should not have general purpose extension methods like this coming from this repo. If there is a need for something like this, open an issue on the configuration repo and drive it there.

Reply:
How will a new method in 2.1.0 help the current WebHooks implementation?

Reply:
Will cover this as part of #178.